packages/ckeditor5-link/src/ui/linkactionsview.ts

```diff
@@ -70,16 +76,20 @@ export default class LinkActionsView extends View {
 
 	private readonly _linkConfig: LinkConfig;
 
+	private readonly _bookmarkEditing: BookmarkEditing | null;
+
 	declare public t: LocaleTranslate;
 
 	/**
 	 * @inheritDoc
 	 */
-	constructor( locale: Locale, linkConfig: LinkConfig = {} ) {
-		super( locale );
+	constructor( editor: Editor, linkConfig: LinkConfig = {} ) {
```

Comment:
This introduces a breaking change. I'd suggest not passing the whole editor instance to the view.

Reply:
Fair point, i've extended the passed arguments list.